app/models/email_address.rb

```diff
@@ -6,6 +6,7 @@ class EmailAddress < ApplicationRecord
   belongs_to :user, inverse_of: :email_addresses
   validates :encrypted_email, presence: true
   validates :email_fingerprint, presence: true
+  has_one :suspended_email, dependent: :destroy
```

Comment:
`dependent: :destroy` means that when this email address is deleted, we destroy the suspend record. I'm not sure what the expected behavior is here? Like I think we want to keep an email suspended until we un-suspend it?

Reply:
Should I remove the dependent destroy then?

Reply:
Let's remove the dependent destroy, we'll probably need to disable a rubocop check

And then let's check back with Team Judy and confirm expected behavior. It shoudn't be possible for a suspended account to delete itself, so I don't think think it's a big risk to merge